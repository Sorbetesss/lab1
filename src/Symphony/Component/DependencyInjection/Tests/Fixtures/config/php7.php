<?php

namespace Symphony\Component\DependencyInjection\Loader\Configurator;

use Symphony\Component\DependencyInjection\Tests\Fixtures\Prototype\Foo;

return function (ContainerConfigurator $c) {
    $c->parameters()
        ('foo', 'Foo')
        ('bar', 'Bar')
    ;
    $c->services()
        (Foo::class)
            ->arg('$bar', ref('bar'))
            ->public()
        ('bar', Foo::class)
            ->call('setFoo')
    ;
};
