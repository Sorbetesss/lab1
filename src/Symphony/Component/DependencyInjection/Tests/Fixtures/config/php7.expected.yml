parameters:
    foo: Foo
    bar: Bar

services:
    service_container:
        class: Symphony\Component\DependencyInjection\ContainerInterface
        public: true
        synthetic: true
    Symphony\Component\DependencyInjection\Tests\Fixtures\Prototype\Foo:
        class: Symphony\Component\DependencyInjection\Tests\Fixtures\Prototype\Foo
        public: true
        arguments: ['@bar']
    bar:
        class: Symphony\Component\DependencyInjection\Tests\Fixtures\Prototype\Foo
        public: true
        calls:
            - [setFoo, {  }]

